### DISCUSSION TITLE
Issues calling wasm support

### DISCUSSION THREAD

**Discussion by see91** [MetaMask/snaps#1886] [Q&A]
I know that snap already supports the use of wasm, and I have successfully run the examples provided in snap.
But when I call my wasm file in snap, I can only get the data of Unit8Array, but not the specific interface method.

I'm not sure if it's because the wasm file content is too large?

The attachment is as follows
[pkg-bundler.zip](https://github.com/MetaMask/snaps/files/13163031/pkg-bundler.zip)

**Comment by Montoya** [comment]
Do you have a repo with your code? I am wary to click a mysterious zip file...

**Reply by see91** [reply]
OK， I have created a repository for this, please check it out

https://github.com/see91/wasm-snap-test

Thanks
